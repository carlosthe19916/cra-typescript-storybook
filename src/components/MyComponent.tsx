import * as React from "react";

export class MyComponent extends React.Component {
  render() {
    return (
      <React.Fragment>
        <p>Hello Lib!</p>
      </React.Fragment>
    );
  }
}
