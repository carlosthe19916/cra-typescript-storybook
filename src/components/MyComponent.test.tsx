import * as React from "react";
import { shallow } from "enzyme";
import { MyComponent } from "./MyComponent";

it("Renders without crashing", () => {
  const wrapper = shallow(<MyComponent />);

  expect(wrapper).toMatchSnapshot();
});
