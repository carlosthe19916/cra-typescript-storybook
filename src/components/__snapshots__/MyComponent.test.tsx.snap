// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without crashing 1`] = `
<Fragment>
  <p>
    Hello Lib!
  </p>
</Fragment>
`;
